src/lib/core.d.ts

```diff
@@ -1150,7 +1150,7 @@ interface ArrayConstructor {
     (arrayLength?: number): any[];
     <T>(arrayLength: number): T[];
     <T>(...items: T[]): T[];
-    isArray(arg: any): boolean;
+    isArray(arg: any): arg is Array<any>;

```

Comment:
@mhegazy I think you're right. Specifying the type is claiming to know something that you don't.